dictionaries/dict.json

```diff
@@ -10,7 +10,7 @@
 "*E/PWOPB/EUBG": "Ebonic",
 "*E/PWOPB/EUBGS": "Ebonics",
 "*E/S*/HREUPBT": "eslint",
-"*E/SP-S": "e",
+"*E": "e",
```

Comment:
For this set of fingerspelled entries in `dict.json`, it might be better to remove them to avoid clashes with glued fingerspelling entries like `"*E": "{>}{&e}"`.

Reply:
This has been changed, but were there any other entries in the set that needed changing?

Reply:
Yes, the rest of the single-letter words in this file, `i` through `w`. Does that make sense?

Reply:
So, I'm interpreting this to read that standard finger-spelled single-letter entries should not be in `dict.json`, and therefore should be removed, to prevent any potential clashes with existing entries in Plover's dictionary. If I'm not barking up the wrong tree and that is the case, then can I assume that entries outside the `i-w` range, like `"KR*": "c"` and `"KWR*": "y"` can also be removed?

Reply:
Yes, @paulfioravanti, your interpretation sounds right to me and `"KR*": "c"`, `"KWR*": "y"` and all the similar entries can be removed including outside the range I mentioned. (I only mentioned `i-w` because of how it looks here in the PR diff.)

Reply:
Okay @didoesdigital, finger-spelled single letter entries have been removed from `dict.json`.